testcafe/runner-photos.js

```diff
@@ -25,6 +30,17 @@ async function runRunner() {
     .run({ assertionTimeout: 6000 }, { pageLoadTimeout: 6000 })
   tc.close()
 
+  //do not try to post to git when using locally
+  if (
+    typeof process.env.TRAVIS_PULL_REQUEST !== 'undefined' &&
+    process.env.TRAVIS_PULL_REQUEST &&
+    process.env.vrErrorMsg != ''
+  ) {
+    const message = `Visual Review - Please review screenshots, then restart build. ${
+      process.env.vrErrorMsg
+    }`
+    await exec(`yarn run cozy-ci-github "${message}"`)
+  }
```

Comment:
I wonder if it's the right place for this — we'll want to do the same for drive, so wouldn't it make sense to have this in a separate script?